package/src/components/CanvasZoom/CanvasZoom.jsx

```diff
@@ -3,19 +3,26 @@ import { fabric } from 'fabric'
 import PropTypes from 'prop-types'
 import { withCanvas } from '@/components/CanvasProvider'
 import { ZOOM_STEP, ZOOM_MIN, ZOOM_MAX } from '@/constants/canvas'
+import { HotKeyEvent } from '@/constants/hotKeys'
+import { withHotKeys } from '@/hocs/withHotKeys'
 import { canvasShape } from '@/models/Canvas'
 
 class CanvasZoom extends Component {
   static propTypes = {
     canvas: canvasShape.isRequired,
     zoom: PropTypes.number,
-    setZoom: PropTypes.func.isRequired
+    setZoom: PropTypes.func.isRequired,
+    registerHandlers: PropTypes.func.isRequired
   }
 
   static defaultProps = {
     zoom: 1
   }
 
+  zoomHotKeyHandler = {
+    [HotKeyEvent.ZOOM]: () => this.onZoom
```

Comment:
you can pass here a 'null' because this is default behaviour of the canvas, no need to add it one more time:
```
[HotKeyEvent.ZOOM]: null
```

Reply:
Hi, fixed